ts/input/tex/base/BaseMethods.ts

```diff
@@ -1221,16 +1221,21 @@ BaseMethods.Cr = function(parser: TexParser, name: string) {
  */
 BaseMethods.CrLaTeX = function(parser: TexParser, name: string, nobrackets: boolean = false) {
   let n: string;
-  if (!nobrackets && parser.string.charAt(parser.i) === '[') {
-    let dim = parser.GetBrackets(name, '');
-    let [value, unit, ] = ParseUtil.matchDimen(dim);
-    // @test Custom Linebreak
-    if (dim && !value) {
-      // @test Dimension Error
-      throw new TexError('BracketMustBeDimension',
-                          'Bracket argument to %1 must be a dimension', parser.currentCS);
+  if (!nobrackets) {
+    if (parser.string.charAt(parser.i) === '*') {  // The * controls page breaking, so ignore it
+      parser.i++;
+    }
+    if (parser.string.charAt(parser.i) === '[') {
```

Comment:
This treatment has always been slightly problematic. Our newline is used in math array structures, so spaces after `\\` would actually have to be treated accordingly. That is, the current treatment is correct for AMS environments but should be ignored in `array` and `eqnarray`. 

In MJ2 base and AMS arrays were treated the same, so this could not really be resolved. However, now they are separated, so we could do that. 
But it's not for this PR to resolve this. Just pointing out that this is an issue waiting to happen.

Reply:
OK, good to know.  I wasn't aware of the distinction.  We can keep it in mind, as you say.